egs/wsj/s5/steps/libs/nnet3/xconfig/lstm.py

```diff
@@ -286,6 +287,10 @@ def check_configs(self):
              raise RuntimeError("dropout-proportion has invalid value {0}."
                                 "".format(self.config['dropout-proportion']))
 
+        if (self.config['dropout-per-frame'] != 'false' and
+            self.config['dropout-per-frame'] != 'true'):
+            raise xparser_error("dropout-per-frame has invalid value {0}.".format(self.config['dropout-per-frame']))
```

Comment:
please change all xparser_errors to  be RuntimeError.  IIRC this exception type has been removed.